tests/performance_tests/performance_test_basic.py

```diff
@@ -602,11 +614,12 @@ def main():
     producerPluginArgs = ProducerPluginArgs(disableSubjectiveBilling=args.disable_subjective_billing,
                                             lastBlockTimeOffsetUs=lbto, produceTimeOffsetUs=args.produce_time_offset_us,
                                             cpuEffortPercent=args.cpu_effort_percent, lastBlockCpuEffortPercent=lbcep,
-                                            producerThreads=args.producer_threads)
-    httpPluginArgs = HttpPluginArgs(httpMaxResponseTimeMs=args.http_max_response_time_ms)
-    netPluginArgs = NetPluginArgs(netThreads=args.net_threads)
+                                            producerThreads=args.producer_threads, maxTransactionTime=-1)
+    httpPluginArgs = HttpPluginArgs(httpMaxBytesInFlightMb=-1, httpMaxResponseTimeMs=-1)
```

Comment:
`httpMaxResponseTimeMs` already has an cli argument and defaults to the normally used large value.  We could leave this alone and still leave the option to override this via cli option.  Otherwise, if you're going to hardcode here you need to remove the `--http-max-response-time-ms` argument.

Reply:
Re-examining our notes, I think I want these as CLI options. I'm also setting the default to -1, unless you think it should just remain a large but limited value.